include/swift/AST/Types.h

```diff
@@ -786,6 +786,10 @@ class alignas(1 << TypeAlignInBits) TypeBase {
   
   /// Check if this is a nominal type defined at the top level of the Swift module
   bool isStdlibType();
+  
+  /// Check if this is either an Array, Set or Dictionary collection type defined
+  /// at the top level of the Swift module
+  bool isStdlibCollectionType();
```

Comment:
@xedin :)
It is ok to do this? Or we could find a better way?

Reply:
IMO a better name would be something like `isKnownStdlibCollectionType`, just to make it clear it won't return true for other stdlib collections like `AnyCollection`, but otherwise this looks good to me :)